icu4j/main/classes/core/src/com/ibm/icu/number/NumberFormatterImpl.java

```diff
@@ -76,40 +89,11 @@ public static int getPrefixSuffixStatic(
         return getPrefixSuffixImpl(microPropsGenerator, signum, output);
     }
 
-    private static final Currency DEFAULT_CURRENCY = Currency.getInstance("XXX");
-
-    final MicroProps micros;
-    final MicroPropsGenerator microPropsGenerator;
-
-    /**
-     * Evaluates the "safe" MicroPropsGenerator created by "fromMacros".
-     */
-    public int format(DecimalQuantity inValue, FormattedStringBuilder outString) {
-        MicroProps micros = preProcess(inValue);
-        int length = writeNumber(micros, inValue, outString, 0);
-        length += writeAffixes(micros, outString, 0, length);
-        return length;
-    }
-
-    /**
-     * Like format(), but saves the result into an output MicroProps without additional processing.
-     */
-    public MicroProps preProcess(DecimalQuantity inValue) {
-        MicroProps micros = microPropsGenerator.processQuantity(inValue);
-        if (micros.integerWidth.maxInt == -1) {
-            inValue.setMinInteger(micros.integerWidth.minInt);
-        } else {
-            inValue.setMinInteger(micros.integerWidth.minInt);
-            inValue.applyMaxInteger(micros.integerWidth.maxInt);
-        }
-        return micros;
-    }
-
```

Comment:
Was this edit intentional? Or is this a side-effect of the messy rebase?

Reply:
Rolled back via d6680d3b1d

Reply:
(We discussed, I heard Shane didn't mind these edits being in this PR - I hope he also doesn't mind me having rolled them back again - wanted to make diff against upstream cleaner!)